server/src/main/java/io/crate/expression/scalar/systeminformation/PgGetExpr.java

```diff
@@ -46,6 +46,16 @@ public static void register(ScalarFunctionModule module) {
             ),
             PgGetExpr::new
         );
+        module.register(
+            Signature.scalar(
+                FQN,
+                DataTypes.STRING.getTypeSignature(),
+                DataTypes.INTEGER.getTypeSignature(),
+                DataTypes.STRING.getTypeSignature(),
+                DataTypes.BOOLEAN.getTypeSignature()
```

Comment:
Last entry is return value - I guess we would need to swap this with previous line

Reply:
oops ... yes